### PR TITLE
Signal implementaiton

### PR DESCRIPTION
**Pull request by PythonMaciej**
* Fix requirements, create new data, first attempt on custom templates

* Add custom view templates for article

Co-authored-by: Jakub Wronka <jakubwronka7@gmail.com>

### CODE REVIEW

**Review by JakubWronka — CHANGES_REQUESTED** [review]
To discuss tomorrow, changes for signals are not visible

File: bestprojectever/blog/signals.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+from .models import Article, CustomUser, CustomUserProfile
+
+@receiver(post_save, sender=CustomUser)
+def create_article_for_new_user(sender, **kwargs):
+    if kwargs['created']:
+        user_number = kwargs['instance'].first_name
+        Article.objects.create(author=kwargs['instance'], title='New user added!', 
+        content='Next user {user} is added'.format(user=user_number))
```
Comment (JakubWronka) [inline bestprojectever/blog/signals.py]: Add newline at the end of the file

**Review by JakubWronka — APPROVED** [review]
Good job overall

File: bestprojectever/blog/signals.py (hunk 1 above)
Comment (PythonMaciej) [reply bestprojectever/blog/signals.py]: done, thank you

**Pull request merged**